prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -537,6 +541,9 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct fi_cq_data_entry *
 	case RXR_DC_EAGER_TAGRTM_PKT:
 		/* completion will be written upon receving the receipt packet, thus no action to be taken her */
 		break;
+	case RXR_DC_EAGER_RTW_PKT:
+		/* no action to be taken here */
+		break;
```

Comment:
Add a similar comment here like "completion will be written upon receving the receipt packet, thus no action to be taken here"

Reply:
OK. Do I need to modify the comment for `RXR_DC_WRITE_RTA_PKT` here? Or I should conform with the comment for `RXR_FETCH_RTA_PKT ` and `RXR_COMPARE_RTA_PKT `